test/test_reactivities_bosch_hale.py

```diff
@@ -0,0 +1,8 @@
+import fusion_toolbox.reaction_rate.bosch_hale as bh
+import numpy.testing as npt
+
+def test_DTn_reactivity():
+    DTn_r= bh.Reactivity("DDp")
+    test_result= 1.3e-17 ## from NRL plasma formulary 
+    test_input = 5.0 # keV
+    npt.assert_almost_equal(DTn_r.get_reactivity(test_input), test_result, decimal=2) 
```

Comment:
```suggestion
    assert DTn_r.get_reactivity(test_input) == pytest.approx(test_result) 
```
You can use the pytest.approx function applied to the expected result and assert they are "equal"

Reply:
Fixed this with 268d8135ec53e82305ac64b1cfea7851caf2b443